src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -29,6 +29,11 @@ object HeaderPattern {
   val hashLineComment = """(?s)((?:#[^\n\r]*(?:\n|\r|\r\n))+(?:\n|\r|\r\n)+)(.*)""".r
 }
 
+object HeaderKeys {
```

Comment:
As a sidenote, I used the same pattern in Gatling's SBT Plugin : https://github.com/gatling/gatling-sbt/blob/master/src/main/scala/io/gatling/sbt/GatlingPlugin.scala